src/core/Form/Select/BaseSelect/SelectItemAddition/SelectItemAddition.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../../../theme';
+import { HtmlDiv, HtmlLi } from '../../../../../reset';
+import { selectItemAdditionStyles } from './SelectItemAddition.baseStyles';
+import classnames from 'classnames';
+
+const baseClassName = 'fi-select-item-addition';
+
+const classNames = {
+  hint_text: `${baseClassName}_hint_text`,
```

Comment:
This naming seems slightly iffy. Is it in line of the general naming convention of these? It's the underscore between hint and text that seems a bit off. Would a hyphen or camelcase be more suitable?

Reply:
True, should use hyphen as separator there. 

Fixed!